lpa_pdb_diagnostics/particles.py

```diff
@@ -839,6 +863,17 @@ def sorted_by_quantity_beam_property ( frame_num, chosen_particles, qdict,
 
                 plt.setp(ax[0].get_xticklabels()[::2], visible=False)
                 plt.setp(ax[1].get_xticklabels()[::2], visible=False)
+                
+            elif b_property == "divergence":
+            
+             	ax.plot( mid_bin, prop*1e6, linewidth = 2 )
+                ax.set_xlabel(r"$\mathrm{%s\,(arb.\, unit)}$"
+                                %quantity_to_analyze)
+                ax.set_ylabel(r"$\mathrm{Divergence}\,(m.\,mrad)}$")
```

Comment:
@kyungtakLIM, change this line to `ax.set_ylabel(r"$\mathrm{Divergence}\,(mrad)}$")`